src/js/stores/AppsStore.js

```diff
@@ -115,6 +116,14 @@ function getAppHealthWeight(health) {
   }, 0);
 }
 
+function getAppType(app) {
```

Comment:
I think some of the filter logic depends on the ``AppTypes``<Array>, thus I would do this in another PR. What do you think?